src/osx/cocoa/dataview.mm

```diff
@@ -2508,6 +2508,19 @@ void UpdateWithRow(int row)
             byExtendingSelection:GetDataViewCtrl()->HasFlag(wxDV_MULTIPLE) ? YES : NO];
 }
 
+void wxCocoaDataViewControl::Select(const wxDataViewItemArray& items)
+{
+    NSMutableIndexSet *selection = [[NSMutableIndexSet alloc] init];
+
+    for (wxDataViewItemArray::const_iterator i = items.begin(); i != items.end(); ++i)
+    {
+        if (i->IsOk())
```

Comment:
Hmm, is it allowed to have invalid items in the selection? Looking at the other ports it doesn't seem so, but they don't explicitly assert in this case neither, so I'm not 100% sure.

Reply:
The generic version does seem to check (but I admit I simply copied this from the single-selection case without thinking too much about it:

```cpp
        int row = m_clientArea->GetRowByItem( item );
        if( row >= 0 )
            m_clientArea->SelectRow(static_cast<unsigned int>(row), true);
```

Reply:
Oops, sorry, I didn't notice this. And after checking wxGTK version, I see that it allows this too. I'd like to add a commit with this test to this branch, so it could run under Mac as well as part of the CI builds -- do you mind?

Reply:
> do you mind?

Not at all (for future reference: that goes implicitly on anything I submit)